docs/ref/models/indexes.txt

```diff
@@ -6,10 +6,12 @@ Model index reference
 
 .. currentmodule:: django.db.models
 
-Index classes ease creating database indexes. They can be added using the
-:attr:`Meta.indexes <django.db.models.Options.indexes>` option. This document
-explains the API references of :class:`Index` which includes the `index
-options`_.
+Index classes ease creating database indexes. They also allow defining various
+other things like type of the index (other than the default one), the index's
+name, the order (ascending/descending) of the index in the database, etc. They
+can be added using the :attr:`Meta.indexes <django.db.models.Options.indexes>`
+option. This document explains the API references of :class:`Index` which
+includes the `index options`_.
```

Comment:
I don't think this block adds much. It's just summarising the options below.

Reply:
I think it is helpful to summarize the main essence of the whole page into a line - particularly for new users who don't know much about indexes (and also for people who are redirected to this page by the `topics/db/optimization.txt` docs which teach users to use indexes for performance benefits).
I think it would also be a good place to redirect people who have the question "why do u think `Meta.indexes` is better than `Meta.index_together`?".

In any case, I have separated the changes into two - feel free to merge whichever make sense to everyone. Thanks for the review.

Reply:
I agree with Carlton.